apps/admin-gui/src/app/shared/components/two-entity-attribute-page/two-entity-attribute-page.component.ts

```diff
@@ -112,14 +122,14 @@ export class TwoEntityAttributePageComponent implements OnInit {
         switch (this.secondEntity){
           case 'member':
             this.resourcesManagerService.getAssignedRichMembers(this.firstEntityId).subscribe(members => {
-              this.entityValues = members;
+              this.entityValues = members.sort(this.compareFnUser);
```

Comment:
when we are sorting the names(first name + last name) of members and users I think we really want to sort according to last name(then according to firstname) + count with czech alphabet